IdentityCore/src/cache/token/MSIDPRTCacheItem.m

```diff
@@ -69,10 +73,12 @@ - (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(NSError **)er
     if ([json msidStringObjectForKey:MSID_SESSION_KEY_CACHE_KEY])
     {
         _sessionKey = [NSData msidDataFromBase64UrlEncodedString:[json msidStringObjectForKey:MSID_SESSION_KEY_CACHE_KEY]];
-        _deviceID = [json msidObjectForKey:MSID_DEVICE_ID_CACHE_KEY ofClass:[NSString class]];
-        _prtProtocolVersion = [json msidObjectForKey:MSID_PRT_PROTOCOL_VERSION_CACHE_KEY ofClass:[NSString class]];
     }
```

Comment:
previously we were setting deviceId and prtocolVersion only if session is present , will this change have any impact ?

Reply:
That's intentional. We want to store deviceId and prt protocol version for new PRTs without sessionKey as well.